CONTRIBUTING.md

```diff
@@ -154,19 +154,34 @@ Common prefixes:
 ### Pull Requests
 - Checkout our [pull request template](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/.github/pull_request_template.md)
 
-#### cpplint
-To see if [__cpplint__](https://github.com/cpplint/cpplint) is already installed, do:
-* `cpplint --version`  # currently returns "cpplint 1.4.4"
-If cpplint is ___not___ installed then do:
-* `python3 -m pip install cpplint`  # If that does not work then try...
-* `py -m pip install cpplint`  # If that does not work then try...
-* `pip install cpplint`
-Once cpplint is installed, test your file(s) with:
-* `cpplint --filter=-legal my_file.cpp my_other_file.cpp`  # Fix any issues and try again.
-
-The [__clang-format__](https://clang.llvm.org/docs/ClangFormat.html) tool can fix whitespace related _cpplint_ issues.
-* On Macs only: `brew install clang-format`  # Only needs to be installed once.
-* All platforms: `clang-format -i -style="{IndentWidth: 4}" my_file.cpp`
+#### Building Locally
+Before submitting a pull request, build the code locally.
```

Comment:
Sorry for the late comments but as I am reading, new thoughts and ideas are coming up and hence suggesting. Thank you for your patience. 
```suggestion
Before submitting a pull request, build the code locally or using [![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus) service.
```